backend/vercel/vercel_test.go

```diff
@@ -82,3 +93,22 @@ func TestCreateLogDrain(t *testing.T) {
 		t.Errorf("failed to create log drain")
 	}
 }
+
+func TestHandleLog(t *testing.T) {
+	server := httptest.NewServer(http.HandlerFunc(HandleLog))
+	defer server.Close()
+
+	body, _ := json.Marshal(&hlog.VercelLog{
+		Message: "hello world",
```

Comment:
Should we assert that this was translated correctly?

Reply:
thanks for the suggestion spenny - was just being lazy not knowing how to set up a mock for the span creation. but once i got that working that found a critical bug in this logic. :tada: